client/src/components/core/ms-input/MsSearchInput.vue

```diff
@@ -37,6 +38,7 @@ defineProps<{
 }>();
 
 const searchRef = ref('');
+const inputRef = ref();
```

Comment:
Not used.

Reply:
It's used [there](https://github.com/Scille/parsec-cloud/pull/6219/files#diff-6b65ea0ed0a55a527b277ba701426036e55d958d41bc0de1d15d42cd3e29f02fR180)